lib/common/common.go

```diff
@@ -60,3 +63,12 @@ func ParseDockerURL(arg string) (*ParsedDockerURL, error) {
 	p, err := docker.ParseDockerURL(arg)
 	return (*ParsedDockerURL)(p), err
 }
+
+// ValidateLayerId validates a layer ID
+func ValidateLayerId(id string) error {
+	validId := regexp.MustCompile(`^(\w+:)?([A-Fa-f0-9]+)$`)
```

Comment:
Sorry, missed this on previous pass: this should a global variable (so failure would be at initialization time, not function invocation time)